Jbarget/oxymore
# Eye interaction project

### What changes have you made?

- added the eye project


### Which issue(s) does this PR fix?

Fixes #15 

### Screenshots (if there are design changes)
![image](https://user-images.githubusercontent.com/11725595/100274876-71a57700-2f5f-11eb-8465-2c78e593df5d.png)



### How to test
- change `launchDate` in `Eye.tsx` to a date in the past
- should see eye data
- should be able to change data set
- if you click on an eye the data should appear in the top RH corner
